content/events/2020-toronto/_index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Toronto
+date: '2020-09-21'
+location:
+  countrycode: ca
+  country: Canada
+  city: Toronto
+  # url:
+  # venue: TBD
+# map: TBD
+# sched_slug: kubernetescommunitydaytoronto TODO
+social:
+  twitter: 'https://twitter.com/KCD_Toronto'
+# sponsors:
```

Comment:
Hello! Great start. Can you please add your three initial sponsors to the webpage? Thanks.